flutter-for-ios.md

```diff
@@ -27,3 +27,1049 @@ that are most relevant to your needs.
+      itemCount: widgets.length,
+      itemBuilder: (BuildContext context, int position) {
+        return getRow(position);
+      });
+
+  Widget getRow(int i) {
+    return new Padding(padding: new EdgeInsets.all(10.0), child: new Text("Row ${widgets[i]["title"]}"));
+  }
+
+  loadData() async {
+    String dataURL = "https://jsonplaceholder.typicode.com/posts";
+    http.Response response = await http.get(dataURL);
+    setState(() {
+      widgets = json.decode(response.body);
+    });
+  }
+}
+{% endprettify %}
+
+# Project Structure & Resources
+
+## How do I include image assets for Flutter? What about multiple resolutions?
+
+While iOS treats images and assets as distinct items, Flutter apps have only
+assets. All your resources which would be placed in the `Images.xcasset`
+folder on iOS should be instead put in an assets folder.
+
+Flutter follows a simple density-based format like iOS. Assets can be `1.0x`,
+`2.0x`, `3.0x`, or any other multiplier. The so-called
+[`devicePixelRatio`](https://docs.flutter.io/flutter/dart-
+ui/Window/devicePixelRatio.html) expresses the ratio of physical pixels in a
+single logical pixel.
+
+Assets on Flutter can be located in any arbitrary folder; there is no
+predefined folder structure. You then declare where the assets are located in
+the `pubspec.yaml` file and Flutter will pick them up.
+
+For example, to add a new image asset called `my_icon.png` to your Flutter
+project, you may decide to store it in a folder we arbitrarily called `images`.
+You would then put the base image (1.0x) in the `images` folder, and all the
+other variants in sub-folders named after the appropriate ratio multiplier:
+ 
+```
+images/my_icon.png       // Base: 1.0x image
+images/2.0x/my_icon.png  // 2.0x image
+images/3.0x/my_icon.png  // 3.0x image
+```
+
+Next, you will need to declare these images in our `pubspec.yaml` file:
+
+<!-- skip -->
+{% prettify yaml %}
+assets:
+ - images/my_icon.jpeg
+{% endprettify %}
+
+You can then access your images using `AssetImage`:
+
+<!-- skip -->
+{% prettify dart %}
+return new AssetImage("images/a_dot_burr.jpeg");
+{% endprettify %}
+
+or directly in an `Image` widget:
+
+<!-- skip -->
+{% prettify dart %}
+@override
+Widget build(BuildContext context) {
+  return new Image.asset("images/my_image.png");
+}
+{% endprettify %}
+
+## Where do I store strings? How do I handle localization?
+
+Unlike iOS, which has the `Localizable.strings` file, Flutter does not
+currently have a dedicated system for handling strings. At the moment, the
+best practice is to declare your copy text in a class as static fields and
+access them from there. For example:
+
+<!-- skip -->
+{% prettify dart %}
+class Strings {
+  static String welcomeMessage = "Welcome To Flutter";
+}
+{% endprettify %}
+
+You can access your strings as such:
+
+<!-- skip -->
+{% prettify dart %}
+new Text(Strings.welcomeMessage)
+{% endprettify %}
+
+
+While there's no functionality built into Flutter to handle
```

Comment:
I've substantially rewritten this section: b9e6371dffeb09244a31b8bbfbf64e0f1e69bce5

If you think it's ok now, I can push the same to the Android doc too, since it has largely the same contents in this section